crates/types/src/message.rs

```diff
@@ -117,6 +122,16 @@ pub enum MessageKind<TYPES: NodeType> {
     External(Vec<u8>),
 }
 
+/// List of keys to send a message to, or broadcast to all known keys
+pub enum RecipientList<K: SignatureKey> {
+    /// Broadcast to all
+    Broadcast,
+    /// Send a message directly to a key
+    Direct(K),
+    /// Send a message directly to many keys
+    Many(BTreeSet<K>),
```

Comment:
is there a reason we don't just use `Vec` here? I guess this comes from the type in `da_broadcast_message`, but it seems a bit weird to have `BTreeSet` in that too

Reply:
It's a bit weird I agree, I had to change this from Vec.  I think using a Set kind of makes sense because we want to ensure no duplicate keys but idk why we'd use the Btree version, we don't need them ordered

Reply:
Oh, the non-duplicates makes sense -- I didn't think of that!